e2e-tests/commands/insurance/account/complete-sign-in-and-go-to-application.js

```diff
@@ -12,11 +12,13 @@ const {
  * Create an account, sign in and create an application via the API or via the full eligibility flow.
  * @param {String} Account email address
  * @param {Boolean} Flag whether to create the application via API instead of going through the eligibility journey. Defaults to true.
+ * @param {String} Company number/companies house number to use for application creation
  * @return {Object} Account ID, application reference number
  */
 const completeSignInAndGoToApplication = ({
   email = mockAccount[EMAIL],
   createApplicationViaApi = true,
+  companyNumber,
```

Comment:
do we need a default company number here?

Reply:
Same as below